OpenRA.Mods.Common/Widgets/Logic/Editor/SaveMapLogic.cs

```diff
@@ -231,11 +231,10 @@ public SaveDirectory(Folder folder, string displayName, MapClassification classi
 			};
 
 			var save = widget.Get<ButtonWidget>("SAVE_BUTTON");
+			save.IsDisabled = () => string.IsNullOrEmpty(filename.Text) || string.IsNullOrEmpty(title.Text) || string.IsNullOrEmpty(author.Text);
```

Comment:
```suggestion
			save.IsDisabled = () => string.IsNullOrWhiteSpace(filename.Text) || string.IsNullOrWhiteSpace(title.Text) || string.IsNullOrWhiteSpace(author.Text);
```
We probably want those to be non-empty as well.